features/taxation/applying_taxes/applying_correct_taxes_for_items_with_the_same_tax_rate.feature

```diff
@@ -40,10 +40,10 @@ Feature: Apply correct taxes for items with the same tax rate
         Then my cart total should be "$73.80"
         And my cart taxes should be "$13.80"
 
-    @ui
+    @ui @api
     Scenario: Proper taxes after changing item quantity
         Given I have 3 products "PHP T-Shirt" in the cart
         And I have 2 products "Symfony Hat" in the cart
-        When I change "PHP T-Shirt" quantity to 1
+        When I change product "PHP T-Shirt" quantity to 1 in my cart
```

Comment:
```suggestion
        When I change the number of "PHP T-Shirt" products in the cart to 1
```
I know we have to clarify, that we are mentioning quantity here, but still what do you think?

Reply:
I'm not convinced as we are using `quantity` keyword both in API and UI for the number of products in the cart, and also in Behat scenarios, so it is clear enough IMO 😃